frank-doc-doclet/src/main/java/org/frankframework/frankdoc/wrapper/FrankMethodDoclet.java

```diff
@@ -51,17 +56,21 @@ public boolean isMultiplyInheritedPlaceholder() {
 
 	@Override
 	public String getName() {
-		return method.name();
+		return method.getSimpleName().toString();
```

Comment:
Zo knullig dat dat nu op deze manier moet 😢

Reply:
Helemaal eens. Een betere `getName()` oid is er niet. Heb ook veel moeite moeten doen om de Generics `<...>` er af te slopen op sommige plekken. Plus dat je niet direct weet dat het er in staat.